### PR TITLE
feat: add function for connecting to Calimero

### PR DESCRIPTION
**Pull request by frdomovic**

### CODE REVIEW

File: src/lib/data/near.js [hunk 1]
```diff
@@ -140,6 +147,7 @@ const getFakKey = async(componentName, near, contract) => {
 
 async function requestFak(componentName, near, contractName, methodNames) {
   const keyPair = nearAPI.utils.KeyPairEd25519.fromRandom();
+
```
Comment (xilosada) [inline src/lib/data/near.js:150]: nit

File: src/lib/data/near.js [hunk 2]
```diff
@@ -184,6 +254,49 @@ const checkFakKey = (rpcResponse, contract, methodNames) => {
...
+      "Method: Near.fakCalimeroCall. Requires requestAccessKey to be called first"
+    );
+  }
+
+  const accountId = await getCurrentAccount(near);
+  const keyStore = new nearAPI.keyStores.InMemoryKeyStore();
+  const keyPair = nearAPI.KeyPair.fromString(key);
+  keyStore.setKey(CalimeroConfig.networkId, accountId, keyPair);
+
+  const calimeroConnection = await nearAPI.connect(
```
Comment (xilosada) [inline src/lib/data/near.js:273]: Can we reuse the Calimero connections in the near object?

File: src/lib/vm/vm.js [hunk 3]
```diff
@@ -806,23 +828,40 @@ class VmStack {
...
+            "Method: Near.call. Required argument: 'contractName'. If the first argument is a string: 'methodName'. Optional: 'args', 'gas' (defaults to 300Tg), 'deposit' (defaults to 0)"
+          );
+        }
+        return this.vm.near.submitFakTransaction(
+          args[0],
+          args[1],
+          args[2] ?? {},
+          args[3],
+          args[4]
+        ); //tu
```
Comment (xilosada) [inline src/lib/vm/vm.js:864]: yo!

**Review by xilosada — APPROVED** [review]
Just some clean up before merge

File: src/lib/data/near.js (hunk 2 above)
Comment (frdomovic) [reply src/lib/data/near.js]: Used existing Calimero connection with updated keystore and signer, tested and works.

**Pull request merged**